### PR TITLE
xfce: update to 4.16

### PR DESCRIPTION
**Pull request by Fearyncess**
Topic Description
-----------------
Update Xfce to version 4.16

Package(s) Affected
-------------------
`extra-xfce/*`
`xfce-base`
`gtksourceview-3`

Security Update?
----------------
No 

Build Order
-----------
`xfce4-dev-tools libxfce4util xfconf libxfce4ui exo garcon gtksourceview-3`

`xfce4-panel xfce4-panel-profiles thunar xfdesktop xfwm4 xfce4-notifyd xfce4-power-manager xfce4-screensaver xfce4-settings xfce4-session mousepad orage tumbler xfce4-appfinder parole ristretto xfburn xfce4-dict xfce4-screenshooter xfce4-taskmanager xfce4-terminal xfdashboard gigolo catfish thunar-volman thunar-archive-plugin thunar-media-tags-plugin thunar-shares-plugin thunar-vcs-plugin`

`xfce4-battery-plugin xfce4-clipman-plugin xfce4-cpufreq-plugin xfce4-cpugraph-plugin xfce4-datetime-plugin xfce4-diskperf-plugin xfce4-eyes-plugin xfce4-fsguard-plugin xfce4-genmon-plugin xfce4-mailwatch-plugin xfce4-mount-plugin xfce4-mpc-plugin xfce4-netload-plugin xfce4-notes-plugin xfce4-pulseaudio-plugin xfce4-sensors-plugin xfce4-smartbookmark-plugin xfce4-systemload-plugin xfce4-time-out-plugin xfce4-timer-plugin xfce4-verve-plugin xfce4-wavelan-plugin xfce4-weather-plugin xfce4-whiskermenu-plugin xfce4-xkb-plugin`

Architectural Progress
----------------------
- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Secondary Architectural Progress
--------------------------------
Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------
- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------
Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-xfce/catfish/spec [hunk 1]
```diff
@@ -1,4 +1,3 @@
-VER=1.4.13
-SRCTBL="https://git.xfce.org/apps/catfish/snapshot/catfish-$VER.tar.gz"
-CHKSUM="sha256::4761a8e4c7fb3df1c0299a8d543d7144e492e88cdcd8fe4b3062ffa43c7c02ad"
-REL=2
+VER=4.15.0
+SRCS="https://archive.xfce.org/src/apps/catfish/${VER%.*}/catfish-$VER.tar.bz2"
+CHKSUMS="whirlpool::858c90d8a56651e1b26c072aa96c4d705d29c43740f88d76f4d7639825a79f93b82f8f38578c8fd82e493345eab2769a908ad7bb7abdf5674c8244f87f970215"
```
Comment (MingcongBai) [inline extra-xfce/catfish/spec:3]: Why whirlpool?

File: extra-xfce/xfce4-power-manager/autobuild/patches/0001-xfce-power-manager-1.6.0-no-mate.patch [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 +++ xfce4-power-manager-1.6.0/settings/xfce4-power-manager-settings.desktop.in	2016-05-26 16:59:29.924613636 -0600
-@@ -8,7 +8,7 @@
- Terminal=false
+@@ -9,7 +9,7 @@
  Type=Application
  Categories=XFCE;GTK;Settings;DesktopSettings;X-XFCE-SettingsDialog;X-XFCE-HardwareSettings;
+ _Keywords=settings;preferences;buttons;sleep;hibernate;battery;suspend;shutdown;brightness;laptop lid;lock screen;plugged in;saving;critical;
 -NotShowIn=GNOME;KDE;Unity;
 +NotShowIn=GNOME;KDE;Unity;MATE;
  StartupNotify=true
```
Comment (MingcongBai) [inline extra-xfce/xfce4-power-manager/autobuild/patches/0001-xfce-power-manager-1.6.0-no-mate.patch:10]: Did you make this patch yourself?

**Review by MingcongBai — APPROVED** [review]

File: extra-xfce/xfce4-power-manager/autobuild/patches/0001-xfce-power-manager-1.6.0-no-mate.patch (hunk 2 above)
Comment (Fearyncess) [reply extra-xfce/xfce4-power-manager/autobuild/patches/0001-xfce-power-manager-1.6.0-no-mate.patch:10]: yes, manually edited from the commit.

File: extra-xfce/catfish/spec (hunk 1 above)
Comment (Fearyncess) [reply extra-xfce/catfish/spec:3]: Faster in the low-preformance devices which have no SHA acceleration with more secure.

**Comment by MingcongBai** [comment]
@fearyncess Reports that Mousepad crashes on Ctrl-F, **DO NOT MERGE**.

**Pull request merged**